### ISSUE TITLE
[Issue] Artithmetic operation overflow.

### ISSUE THREAD

**Issue by ptc0** [workspacer/workspacer#511]
hello workspacer developers/contribuiters!
i am getting this error everytime i launch the program, and i thought i should let you guys know!

```
Unhandled exception. System.OverflowException: Arithmetic operation resulted in an overflow.
   at System.IntPtr.op_Explicit(IntPtr value)
   at workspacer.ConsoleHelper.get_IsConsoleShowing()
   at workspacer.ConsoleHelper.set_IsConsoleShowing(Boolean value)
   at workspacer.ConsoleHelper.Initialize()
   at workspacer.Watcher.Program.Main(String[] args)
```

it seems to be an overflow, i belive its an value that was set that was too large or too small for the int limit (probably a too large value).
cheers! :)

**Comment by ptc0** [comment]
i think the issue is being related at the file in: `./src/workspacer.Watcher/ConsoleHelper.cs`, the problem may be in `get_IsConsoleShowing()` method in the ConsoleHelper class, its causing the overflow when is converting from IntPtr to another data type.

note: im new to csharp and i may be wrong about it

**Comment by ethanrutt** [comment]
I am hitting this exact same error as well. Fresh install through `choco` with nothing added.

**Comment by korbajan** [comment]
I'm getting the same error.

**Comment by ptc0** [comment]
i have modified some settings in the workspacer.config.csx file, but i was getting the same error when i installed it in the first time, i think ill try to reinstall it via `choco` and let you guys know

**Comment by animeshchaudhri** [comment]
I'm getting the same issue

**Comment by josteink** [comment]
I'm not really active as an maintainer anymore, but if anyone cranks up a PR fixing this issue, I'll be sure to get it merged.

**Comment by ethanavatar** [comment]
This issue appears to be fixed in the unstable version. I'm using [0.9.12.196](https://github.com/workspacer/workspacer/releases/tag/unstable)

**Comment by Aareon** [comment]
Can anyone else confirm that this issue has been resolved by [>=0.9.12.196](https://github.com/workspacer/workspacer/releases/tag/unstable)

**Comment by ptc0** [comment]
fixed

**Comment by derekthecool** [comment]
Fixed for me as well using version 0.9.12.196! Yay!
